### PR TITLE
Added test case to make sure when teamd feature/service is stop kernel device is cleanup and SIGTERM are handled gracefully

### PR DESCRIPTION
**Pull request by abdosi**
Why/What I did:-
Added test case to make sure when teamd feature/service is stop 
portchannel are cleanup in kernel and we get syslog
from teammgrd and teamsyncd process of handling
SIGTERM signal

This PR is raise to track the fixes done by PR-
1) https://github.com/Azure/sonic-swss/pull/1407
2) https://github.com/Azure/sonic-swss/pull/1159

How I verify:

sudo py.test --inventory "../ansible/str" --host-pattern str-s6000-on-2 --module-path "../ansible/library/" --testbed vms13-5-t1-lag --testbed_file "../ansible/testbed.csv"  -vvv pc/test_po_cleanup.py

On KVM VS Switch:
sudo py.test --inventory "veos.vtb" --host-pattern vlab-01 --module-path "../ansible/library/" --testbed vms-kvm-t0 --testbed_file "vtestbed.csv"  -vvv pc/test_po_cleanup.py --skip_sanity

Currently this test case will fail on master and 201911 as we are waiting for PR#https://github.com/Azure/sonic-swss/pull/1407 to be merged.

Update:

With PR#1407 test case is passing:

sudo py.test --inventory "../ansible/str" --host-pattern str-a7050-acs-1 --module-path "../ansible/library/" --testbed vms2-5-t0-7050-1  --testbed_file
"../ansible/testbed.csv"  -vvv pc/test_po_cleanup.py --skip_sanity
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.12, pytest-4.6.5, py-1.8.1, pluggy-0.13.1 -- /usr/bin/python
cachedir: .pytest_cache
ansible: 2.8.7
rootdir: /data/Networking-acs-sonic-mgmt/tests, inifile: pytest.ini
plugins: ansible-2.2.2, xdist-1.28.0, repeat-0.8.0, forked-1.1.3
collected 1 item

pc/test_po_cleanup.py::test_po_cleanup PASSED                                                                                                                                                               [100%]

=========================================================================================== 1 passed in 210.76 seconds ============================================================================================

### CODE REVIEW

File: tests/pc/test_po_cleanup.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            ".*ERR swss#orchagent: :- removeLag.*",
+        ]
+        loganalyzer.ignore_regex.extend(ignoreRegex)
+        expectRegex = [
+            ".*teamd#teammgrd: :- cleanTeamProcesses.*",
+            ".*teamd#teamsyncd: :- cleanTeamSync.*"
+        ]
+        loganalyzer.expect_regex.extend(expectRegex)
+
+    yield
```
Comment (yxieca) [inline tests/pc/test_po_cleanup.py:35]: I don't think you need to add yield if no clean up is required

File: tests/pc/test_po_cleanup.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+
+def check_kernel_po_interface_cleaned(duthost):
+    res = duthost.shell("ip link show | grep -c PortChannel | cat")["stdout_lines"][0].decode("utf-8")
```
Comment (yxieca) [inline tests/pc/test_po_cleanup.py:39]: What does this pipe to cat do?

Comment (yxieca) [reply tests/pc/test_po_cleanup.py:39]: Are you trying to mute the return errorcode? you can use command_output = self.shell(cmd, module_ignore_errors=True) to ignore the command error code instead.

Comment (abdosi) [reply tests/pc/test_po_cleanup.py:39]: @yxieca yes, that was the reason for using cat. with output as "0" ansible module restrn error.
I will update to use module_ignore_errors as True

File: tests/pc/test_po_cleanup.py (hunk 1 above)
Comment (abdosi) [reply tests/pc/test_po_cleanup.py]: @yxieca Done

File: tests/pc/test_po_cleanup.py (hunk 2 above)
Comment (abdosi) [reply tests/pc/test_po_cleanup.py]: @yxieca Done

**Review by yxieca — APPROVED** [review]

**Review by pavel-shirshov — APPROVED** [review]
LGTM

**Pull request merged**